### ISSUE TITLE
Make ToggleSwitch accessible

### ISSUE THREAD

**Issue by dmytro-danylov** [wix/wix-style-react#1235]
### Summary:
We need to make the ToggleSwitch component accessible by keyboard. Also, screen readers should describe it properly when a user focuses on it.

### Priority:
- [  ] Blocker
- [ ] Urgent
- [  ] Normal
- [  ] Minor

### Steps to reproduce:
The component is not being focused when pressing the "TAB" key.

### Expected behavior:
The component should be focused visually and properly treated by screen readers. Screen readers should describe where the checkbox belongs and give feedback to the user whether it's checked or not. When component is focused you should be able to toggle it by pressing the spacebar. When user presses the spacebar, the screen reader should say whether he/she checked or unchecked it.

**Comment by shlomitc** [comment]
ToggleSwitch is accessible now. closing, please re-open if there are any issues
